app/helpers/style_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module StyleHelper
+  def stylesheet_for_layout
+    if asset_exist?'custom.css'
+      'custom'
+    else
+      'application'
+    end
```

Comment:
`asset_exist? 'custom.css' ? 'custom' : 'application'` might be simpler here. Otherwise, looking good.